.github/workflows/check-build-deploy.yaml

```
@@ -101,6 +101,9 @@ jobs:
             - name: Run pre-commit
               run: uv run -- pre-commit run --all-files --hook-stage manual  # TODO: Add GitHub workflows output format
 
+            - uses: pre-commit-ci/lite-action@v1.1.0
+              if: always()
```

```suggestion
              if: ${{ !cancelled() }}
```
See the warning at https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always